docs/eventing/sinks/kafka-sink.md

```diff
@@ -86,7 +86,7 @@ spec:
   # - structured
   # - binary
   #
-  # default: structured.
+  # default: binary.
```

Comment:
Perhaps giving a primer on these 2 modes would be helpful as an addition in this page

Reply:
+1 @matzew can you provide some background information in a follow up issue or PR please